spk/rutorrent/src/bin/rtorrent-daemon

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+# Package
+PACKAGE="rutorrent"
+
+INSTALL_DIR="/var/packages/${PACKAGE}/target"
+WEB_DIR="/var/services/web"
+RTORRENT_RC=${WEB_DIR}/${PACKAGE}/conf/rtorrent.rc
+
+export HOME=${INSTALL_DIR}/var
+export PATH="${INSTALL_DIR}/env/bin:${INSTALL_DIR}/bin:${INSTALL_DIR}/usr/bin:${PATH}"
+export LD_LIBRARY_PATH="${INSTALL_DIR}/lib:${LD_LIBRARY_PATH}"
+RTORRENT="${INSTALL_DIR}/bin/rtorrent"
+
+# If default config ($HOME/.rtorrent.rc) is a symlink
+# and package default location file exists, use parameters
+# to force using that config file
+if [ -L ${HOME}/.rtorrent.rc -a -f ${RTORRENT_RC} ]; then
```

Comment:
Maybe a stupid question, but why having this `.rtorrent.rc` symlink file hanging around anyway ? If you can use the configuration file directly ? It would simplify a lot the logic here, wouldn't it ?

Reply:
I mean we could just get rid of that file, right ?

Reply:
Indeed we could... and ain't sure what to do about it.

By default `rtorrent` looks into the `$HOME` directory.  I thought one might prefer having another copy instead of using the package default... I noticed that it does provide a sort of failsafe when things turns sideways.  Worth keeping?  honestly no clue.  Worth the effort to modify, again?

Reply:
Honestly, I tend to like the current behavior... and I am too lazy to do the change (as I think we spent to many hours on this package).  I'd rather just move on and go ahead with the merge & distributing, and perhaps, revisit some other time.  But feel free to adjust as you see fit :)